apex/python/log.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import time
+
+class message:
+    def __init__(self):
+        self.start_time = time.time()
+        logging.basicConfig(filename="apex_tacos.log", filemode="w", format="%(name)s - %(levelname)s - %(message)s", level=logging.INFO)
```

Comment:
Don't use the root logger. A message or logger class should have its own logger. From here, create a handler (File, console etc) and set the format, level etc